quarkus/tests/integration/src/main/java/org/keycloak/it/junit5/extension/CLIResult.java

```diff
@@ -92,6 +92,10 @@ default void assertMessage(String message) {
         assertThat(getOutput(), containsString(message));
     }
 
+    default void assertNoMessage(String message) {
+        assertFalse(getOutput().contains(message));
```

Comment:
small thing I noticed: pls use `assertThat(getOutput(), containsString(message));` instead of assertFalse, as e.g. in line 92 above, this gives better test output when something goes wrong (instead of "expected: true, but was: false" it shows the actual string that is (not) contained in the errmsg .